### DISCUSSION TITLE
Verrouiller un PDF produit par FRW

### DISCUSSION THREAD

**Discussion by jpbeaulieu1** [MTESSDev/FRW#248] [Annonces]
Bonjour

Est-ce qu'il y a une configuration qui indique à FRW de verrouiller les PDF générés pour éviter leurs modification? Surtout si on envoie ce PDF en pièce jointe dans un courriel.

![image](https://github.com/MTESSDev/FRW/assets/112179879/2b9ac513-3b77-4b78-96cf-52c4968562a0)

**Comment by Hutson48** [comment]
Bonjour,

Actuellement, il n’existe pas de configuration dans FRW permettant de verrouiller les PDF générés pour empêcher leur modification, même lorsqu’ils sont envoyés en pièce jointe par courriel. Il est possible que vous deviez utiliser une solution externe pour sécuriser ou protéger les documents PDF après leur génération. 

Si cela est important pour vous, vous pouvez ouvrir une issue et je la prioriserai éventuellement. 

Bonne journée,
